candi/openapi/node_group.yaml

```diff
@@ -501,6 +501,15 @@ spec:
                       type: string
                       x-doc-default: '/var/lib/kubelet'
                       description: Directory path for managing kubelet files (volume mounts,etc).
+                    containerLogMaxSize:
```

Comment:
How does it work with docker? I mean, if rotation is enabled in both places: docker and kubelet.

Reply:
Docker is deprecated. This change focuses the mind of an engineer on a crucial task: migrating away from docker ASAP.

Reply:
I wonder about the impact. How setting both docker and kubelet log rotation options will affect the system? 

I assume that kubelet logs rotation will have no effect in the case of having Docker as a CRI. If so, the only thing we need to do is add a warning about it to the description.

Reply:
Here's the relevant [check](https://github.com/kubernetes/kubernetes/blob/8d456f2b9138771b90bb17ec5cdcb4f9a11c4243/pkg/kubelet/kubelet.go#L632). It won't even start rotating logs on docker.

I'll update the description ASAP.